src/Microsoft.Azure.SignalR.Management/Microsoft.Azure.SignalR.Management.csproj

```diff
@@ -1,6 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <RootNamespace>Microsoft.Azure.SignalR.Management</RootNamespace>
+    <Description>Provides communication management capabilities for Azure SignalR.</Description>
```

Comment:
Provide a way to communicate with connected clients directly through Azure SignalR service, including broadcasting to clients .....